projects/assets/src/translations/en/my-account.ts

```diff
@@ -36,14 +36,14 @@ export const myAccount = {
       returnAction: 'Request a Return',
       cancelAction: 'Cancel Order Items',
       item: 'Item',
-      itemPrice: 'Price',
-      quantity: 'Qty',
-      returnQty: 'Return Qty',
-      cancelQty: 'Cancel Qty',
-      returnAll: 'Return All',
-      cancelAll: 'Cancel All',
+      itemPrice: 'Item Price',
+      quantity: 'Max Quantity',
+      returnQty: 'Quantity to Return',
+      cancelQty: 'Quantity to Cancel',
+      setAll: 'Set all quantities to maximum',
       totalPrice: 'Total',
       submit: 'Submit Request',
+      returnNote: 'The following items will be included in the return request',
```

Comment:
Return complete order?

Reply:
No, it just a note displayed in return confirmation page.